QUICKSTART-ECS.md

```diff
@@ -0,0 +1,191 @@
+That either means adding `--associate-public-ip-address` to the launch command below, or attaching an Elastic IP address after launch.
+There will be a reminder about this when we talk about the launch command.
+
+Finally, note that if you want to launch in a specific availability zone, make sure you pick a subnet that matches; the AZ is listed right below the public/private status.
+
+### IAM role
+
+The instance we launch needs to be associated with an IAM role that allows for communication with ECS.
+
+ECS provides a
+[managed policy](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_managed_policies.html#AmazonEC2ContainerServiceforEC2Role)
+with all of the appropriate permissions.
+If you've used ECS before, you may already have an appropriate role in your account called `ecsInstanceRole`.
+If you do not, you can
+[follow the instructions in the ECS Developer Guide to create a role](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/instance_IAM_role.html).
+
+Note down the instance role name in your account for the instructions below.
+
+#### Enabling SSM
+
+If you add SSM permissions, you can use Bottlerocket's default SSM agent to get a shell session on the instance.
+
+To attach the role policy for SSM permissions, run the following (replacing INSTANCE_ROLE_NAME with the name of your instance role):
+
+```
+aws iam attach-role-policy \
+   --role-name INSTANCE_ROLE_NAME \
+   --policy-arn arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+```
+
+Next, to retrieve the instance profile name used to launch instances, run this:
+
+```
+aws iam list-instance-profiles-for-role --role-name INSTANCE_ROLE_NAME --query "InstanceProfiles[*].InstanceProfileName" --output text
+```
+
+Note this down as the INSTANCE_PROFILE_NAME for the final launch command.
+
+### Connecting to your cluster
+
+For the instance to be able to communicate with ECS, we need to make sure to configure the instance with the name of the cluster.
+
+Create a file called `userdata.toml` with the following contents, where CLUSTER_NAME is the name of the cluster you created above (for example, "bottlerocket").
+
+```
+[settings.ecs]
+cluster = "CLUSTER_NAME"
+```
+
+If you want to customize the behavior of your instance further, you can find the full set of supported settings [here](README.md#settings).
+
+### Launch!
+
+Now we can launch a Bottlerocket instance in our cluster!
+
+There are a few values to make sure you change in this command:
+* YOUR_KEY_NAME: your SSH keypair name, as registered with EC2
+* SUBNET_ID: the subnet you selected earlier
+  * If you chose a public subnet, either add `--associate-public-ip-address` to the command, or attach an Elastic IP afterward.
+* BOTTLEROCKET_AMI_ID: the Amazon-provided AMI ID you found above
+* userdata.toml: the path to the user data file you created earlier
+* INSTANCE_PROFILE_NAME: the IAM instance profile you created
```

Comment:
```suggestion
* INSTANCE_PROFILE_NAME: the IAM instance profile you created, e.g. `ecsInstanceRole`
```
This is going to be the case for almost everyone, so I don't think it hurts to reinforce this here

Reply:
Done